### ISSUE TITLE
Can't Fast Forward nor change speed.

### ISSUE THREAD

**Issue by asdfghkrrrr** [TASEmulators/BizHawk#4031]
I've already tried to disable VSync or delete the .ini files, but nothing seems to work. I press tab but nothing changes. Same applies when I change speed. Help me pls

**Comment by YoshiRulz** [comment]
What version do you have? What game are you using?

**Comment by CasualPokePlayer** [comment]
Are you sure the GPU driver settings aren't just enforcing VSync?

**Comment by Morilli** [comment]
As commented above, this is most likely being caused by GPU settings either forcing v-sync or a maximum framerate. Closing due to non-reproducability and lack of response.

**Comment by AxelXtreme** [comment]
I'm also having this problem where fast forward doesn't work. I saw the comments about the GPU forcing VSync and so I changed that and it worked! But now it stopped working and I don't know why.

I'm using version 2.10 of Bizhawk with the Display Method as Direct3D11, Speed as 100%, Alternate Speed as 400%, Auto-minimize skipping and Skip 4.

My OS is Windows 11 and my GPU is NVIDIA GeForce RTX 3060.
